nixos/modules/hardware/xone.nix

```diff
@@ -21,6 +21,7 @@ in
       extraModulePackages = with config.boot.kernelPackages; [ xone ];
     };
     hardware.firmware = [ pkgs.xow_dongle-firmware ];
+    hardware.xpad-noone.enable = lib.mkDefault true;
```

Comment:
Please add a release notes entry for this change

Reply:
Added